osf/migrations/0142_add_spam_fields_to_user.py

```diff
@@ -17,12 +18,13 @@
 def add_spam_status_to_tagged_users(state, schema):
     OSFUser = state.get_model('osf', 'osfuser')
     users_with_tag = OSFUser.objects.filter(tags__name__in=TAG_MAP.keys()).prefetch_related('tags')
+    users_to_update = []
     for user in users_with_tag:
         for tag, value in TAG_MAP.items():
-            if user.tags.filter(name=tag).exists():
+            if user.tags.filter(system=True, name=tag).exists():
                 user.spam_status = value
-
-        user.save()
+        users_to_update.append(user)
+    bulk_update(users_to_update)
```

Comment:
I'd use the update_fields parameter, I think it's something like, 
```python
bulk_update(users_to_update, update_fields=['spam_status'])
```

There's also a batch_size parameter, that can split the update into chunks.